templates/admin/base_site.html

```diff
@@ -1,7 +1,15 @@
-{% extends 'admin/base.html' %}
-{% load staticfiles rev %}
+{% extends "admin/base.html" %}
+{% load i18n rev %}
+
+{% block title %}{{ title }} | {% trans "Django girls admin" %}{% endblock %}
```

Comment:
Can we make `g` a capital letter `G` in `girls`?

Reply:
@amakarudze Done 👍